plugins/make_plugin_rpm

```diff
@@ -190,7 +190,11 @@ if [ -d ${GIT_ROOT}/python ]; then
 	find ${GIT_ROOT}/python | grep -E "(__pycache__|\.pyc|\.pyo$)" | xargs rm -rf
 	cp -R ${GIT_ROOT}/python .
 else
-	(cd ${GIT_ROOT}; mkdir -p build; cd build; cmake ..; make)
+	(cd ${GIT_ROOT}; 
+		if [ -f requirements.sh ]; then
+			sh requirements.sh
```

Comment:
We need another exercise to update `requirements.sh` for All repos example ([opcua](https://github.com/foglamp/foglamp-south-opcua/blob/develop/requirements.sh))
`apt install` Vs `yum install`